### PR TITLE
Update TW Translation

### PR DESCRIPTION
**Pull request by NeKoOuO**
Fix typos, add translation

### CODE REVIEW

**Comment by FooIbar** [comment]
Do you really call gallery "相簿" in TW?
That does not make sense to me.
It seems "畫廊" is more commonly used.
https://ehwiki.org/wiki/Galleries/Chinese
https://www.ptt.cc/bbs/AC_In/M.1584886579.A.F5A.html

**Comment by FooIbar** [comment]
@5idereal Since you've done this previously, would you mind taking a look?

**Comment by revonateB0T** [comment]
`New Compose Reader` is added for developers.
This option will be removed and users are forced to use Compose one after it's completed.

**Comment by revonateB0T** [comment]
> @5idereal Since you've done this previously, would you mind taking a look?

cc @bluehomewu

**Comment by NeKoOuO** [comment]
Done

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <string name="text_is_empty">文字是空的</string>
     <string name="waring">警告</string>
     <string name="invalid_download_location">看來下載路徑似乎沒辦法使用。請至設定重新設定下載路徑。</string>
-    <string name="clipboard_gallery_url_snack_message">剪貼簿裡有圖庫連結</string>
+    <string name="clipboard_gallery_url_snack_message">剪貼簿裡有畫廊連結</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:31]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 2]
```diff
@@ -53,7 +53,7 @@
     <!-- Gallery List -->
     <string name="gallery_list_search_bar_hint_exhentai">在 ExHentai 搜尋</string>
     <string name="gallery_list_search_bar_hint_e_hentai">在 E-Hentai 搜尋</string>
-    <string name="gallery_list_search_bar_open_gallery">開啟圖庫</string>
+    <string name="gallery_list_search_bar_open_gallery">開啟畫廊</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:56]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 3]
```diff
@@ -75,11 +75,11 @@
     <string name="remove_from_favourites">移除收藏</string>
     <string name="delete_downloads">刪除下載</string>
     <string name="quick_search">快速搜尋</string>
-    <string name="quick_search_tip">點擊“+”以新增快速搜尋</string>
+    <string name="quick_search_tip">點選“+”以新增快速搜尋</string>
     <string name="add_quick_search_dialog_title">新增快速搜尋</string>
     <string name="name_is_empty">名稱是空的</string>
     <string name="delete">刪除</string>
-    <string name="add_quick_search_tip">圖庫列表的狀態將被儲存為快速搜尋。如果你想儲存搜尋面板的狀態，請先進行搜尋。</string>
+    <string name="add_quick_search_tip">畫廊列表的狀態將被儲存為快速搜尋。如果你想儲存搜尋面板的狀態，請先進行搜尋。</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:82]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 4]
```diff
@@ -101,7 +101,7 @@
     <string name="torrent_count">種子 (%d)</string>
     <string name="share">分享</string>
     <string name="rate">評分</string>
-    <string name="similar_gallery">類似圖庫</string>
+    <string name="similar_gallery">類似畫廊</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:104]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 5]
```diff
@@ -114,11 +114,12 @@
     <string name="refresh_original">重新整理 (原圖)</string>
     <string name="open_in_other_app">在其他應用程式中開啟</string>
     <string name="clear_image_cache">清除圖片快取</string>
-    <string name="clear_image_cache_confirm">為該圖庫清除所有快取圖片？</string>
+    <string name="clear_image_cache_confirm">為該畫廊清除所有快取圖片？</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:117]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 6]
```diff
@@ -114,11 +114,12 @@
...
     <string name="image_cache_cleared">已清除圖片快取</string>
     <string name="rate_successfully">評分成功</string>
     <string name="rate_failed">評分失敗</string>
     <string name="no_torrents">沒有種子</string>
+      <string name="no_current_torrents">沒有當前版本畫廊的種子</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:122]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 7]
```diff
@@ -131,7 +132,7 @@
     <string name="filter_the_uploader">把上傳者“%s”加入過濾列表？</string>
     <string name="filter_the_tag">隱藏標籤“%s”？</string>
     <string name="filter_added">已經新增篩選條件</string>
-    <string name="newer_version_available">此圖庫有新版本可以使用。</string>
+    <string name="newer_version_available">此畫廊有新版本可以使用。</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:135]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 8]
```diff
@@ -146,7 +147,7 @@
     <string name="rating1">快要窒息了</string>
     <string name="rating0">下去！</string>
     <!-- Gallery Info -->
-    <string name="gallery_info">圖庫資訊</string>
+    <string name="gallery_info">畫廊資訊</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:150]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 9]
```diff
@@ -157,7 +158,7 @@
     <string name="key_category">分類</string>
     <string name="key_uploader">上傳者</string>
     <string name="key_posted">上傳時間</string>
-    <string name="key_parent">上層圖庫</string>
+    <string name="key_parent">上層畫廊</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:161]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 10]
```diff
@@ -169,7 +170,7 @@
     <string name="key_torrents">種子數</string>
     <string name="key_torrent_url">種子連結</string>
     <!-- Gallery Comments -->
-    <string name="gallery_comments">圖庫留言</string>
+    <string name="gallery_comments">畫廊留言</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:173]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 11]
```diff
@@ -197,7 +198,7 @@
     <string name="format_url">連結</string>
     <string name="format_plain">純文字</string>
     <!-- Gallery Previews -->
-    <string name="gallery_previews">圖庫預覽</string>
+    <string name="gallery_previews">畫廊預覽</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:201]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 12]
```diff
@@ -228,13 +229,17 @@
     <string name="download_start_all">全部開始</string>
     <string name="download_stop_all">全部停止</string>
     <string name="download_reset_reading_progress">重置閱讀進度</string>
-    <string name="reset_reading_progress_message">重置所有已下載圖庫的閱讀進度？</string>
+    <string name="reset_reading_progress_message">重置所有已下載畫廊的閱讀進度？</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:232]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 13]
```diff
@@ -248,7 +253,7 @@
     <string name="clear_all">全部清除</string>
     <string name="clear_all_history">清除所有歷史紀錄？</string>
     <!-- Filter -->
-    <string name="filter">隱藏圖庫</string>
+    <string name="filter">隱藏畫廊</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:256]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 14]
```diff
@@ -259,7 +264,7 @@
     <string name="add_filter">新增隱藏項</string>
     <string name="show_definition">檢視定義</string>
     <string name="filter_text">隱藏項文字</string>
-    <string name="filter_tip">該隱藏系統會在 EHentai 網站隱藏系統的基礎上繼續隱藏圖庫。\n\n標題隱藏項：排除標題含有該關鍵字的圖庫。\n\n上傳者隱藏項：排除該上傳者上傳的圖庫。\n\n標籤隱藏項：排除包含該標籤的圖庫，這會使獲取圖庫列表花費更多時間。\n\n標籤組隱藏項：排除包含該標籤組的圖庫，這會使獲取圖庫列表花費更多時間。</string>
+    <string name="filter_tip">該隱藏系統會在 EHentai 網站隱藏系統的基礎上繼續隱藏畫廊。\n\n標題隱藏項：排除標題含有該關鍵字的畫廊。\n\n上傳者隱藏項：排除該上傳者上傳的畫廊。\n\n標籤隱藏項：排除包含該標籤的畫廊，這會使獲取畫廊列表花費更多時間。\n\n標籤組隱藏項：排除包含該標籤組的畫廊，這會使獲取畫廊列表花費更多時間。</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:267]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 15]
```diff
@@ -280,7 +285,7 @@
     <string name="settings_u_config_summary">EHentai 網站上的設定</string>
     <string name="settings_my_tags">我的標籤</string>
     <string name="settings_my_tags_summary">EHentai 網站上的我的標籤</string>
-    <string name="settings_eh_gallery_site">圖庫站台</string>
+    <string name="settings_eh_gallery_site">畫廊站臺</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:288]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 16]
```diff
@@ -296,8 +301,8 @@
     <string name="settings_eh_force_eh_thumb_summary">若exhentai載入遇到問題可嘗試此項\n注意，此項將使用 e-hentai 設定來載入圖片。</string>
     <string name="settings_eh_show_jpn_title">顯示日文標題</string>
     <string name="settings_eh_show_jpn_title_summary">需同時在 EHentai 網站設定中啟用 Japanese Title</string>
-    <string name="settings_eh_show_gallery_pages">顯示圖庫頁數</string>
-    <string name="settings_eh_show_gallery_pages_summary">在圖庫列表中顯示頁數</string>
+    <string name="settings_eh_show_gallery_pages">顯示畫廊頁數</string>
+    <string name="settings_eh_show_gallery_pages_summary">在畫廊列表中顯示頁數</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:305]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 17]
```diff
@@ -296,8 +301,8 @@
...
     <string name="settings_eh_show_gallery_comments">顯示畫廊評論</string>
     <string name="settings_eh_show_gallery_comments_summary">在畫廊詳情頁中顯示評論</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:307]: Don't forget.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 18]
```diff
@@ -306,8 +311,8 @@
     <string name="settings_eh_show_tag_translations_summary">顯示翻譯後的標籤而非原始文字（需花費時間來下載資料檔案）</string>
     <string name="settings_eh_tag_translations_source">補充翻譯（由 EhTagTranslation 提供）</string>
     <string name="settings_eh_tag_translations_source_url">https://github.com/EhTagTranslation/Editor/wiki</string>
-    <string name="settings_eh_filter">隱藏圖庫</string>
-    <string name="settings_eh_filter_summary">根據標題、上傳者、標籤、评论者隱藏圖庫或评论</string>
+    <string name="settings_eh_filter">隱藏畫廊</string>
+    <string name="settings_eh_filter_summary">根據標題、上傳者、標籤、評論者隱藏畫廊或評論</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:315]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 19]
```diff
@@ -324,9 +329,13 @@
...
     <string name="settings_download_archive_metadata_summary">下載壓縮檔時產生 ComicInfo.xml</string>
+    <string name="settings_download_reload_metadata">重新載入中繼資料</string>
+    <string name="settings_download_reload_metadata_summary">為標籤可能發生變動的下載項更新 ComicInfo.xml</string>
+    <string name="settings_download_reload_metadata_successfully">成功載入 %d 項</string>
+    <string name="settings_download_reload_metadata_failed">載入中繼資料失敗: %s</string>
     <string name="settings_download_media_scan">允許其他程式進行媒體掃描</string>
-    <string name="settings_download_media_scan_summary_on">請別讓別人看到你的相簿</string>
-    <string name="settings_download_media_scan_summary_off">大部分相簿軟體將會忽略顯示下載路徑中的影像</string>
+    <string name="settings_download_media_scan_summary_on">請別讓別人看到你的畫廊</string>
+    <string name="settings_download_media_scan_summary_off">大部分畫廊軟體將會忽略顯示下載路徑中的影像</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:338]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 20]
```diff
@@ -420,9 +429,10 @@
     <string name="clear_search_history_confirm">清除搜尋記錄？</string>
     <string name="search_history_cleared">已清除搜尋記錄</string>
     <string name="download_service">下載服務</string>
-    <string name="favorite_name">收藏圖庫</string>
+    <string name="favorite_name">收藏畫廊</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:432]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 21]
```diff
@@ -460,34 +470,36 @@
...
     <string name="update_frequency_biweekly">每兩週</string>
     <string name="update_frequency_monthly">每月</string>
-    <string name="update_failed">更新失败：%s</string>
+    <string name="update_failed">更新失敗：%s</string>
     <string name="new_version_available">有新版本可用！</string>
     <string name="already_latest_version">已是最新版本</string>
-    <string name="permission_denied">沒有權限</string>
-    <string name="download_gallery_first">請先下載圖庫！</string>
+    <string name="permission_denied">沒有許可權</string>
+    <string name="download_gallery_first">請先下載畫廊！</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:502]: We use "相簿" or "圖庫" as gallery in Taiwan.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 22]
```diff
@@ -460,34 +470,36 @@
...
+    <string name="filter_label">隱藏項型別</string>
     <string name="archive_passwd">壓縮檔密碼</string>
     <string name="archive_need_passwd">壓縮檔需要密碼</string>
     <string name="passwd_wrong">密碼錯誤</string>
     <string name="passwd_cannot_be_empty">密碼不能為空</string>
     <string name="list_tile_thumb_size">詳情模式下縮圖大小</string>
     <string name="account_name">帳號</string>
     <string name="preload_thumb_aggressively">積極地預載入縮圖</string>
+    <string name="animate_items">列表專案動畫</string>
+    <string name="animate_items_summary">如遇崩潰/掉幀請嘗試禁用此項</string>
```
Comment (bluehomewu) [inline app/src/main/res/values-zh-rTW/strings.xml:490]: "停用" is better than "禁用", "禁用" is used in China.

**Review by bluehomewu — COMMENTED** [review]
The suggested items have been replied line by line. Please make corrections for each item.

**Comment by FooIbar** [comment]
Disclaimer: I'm not Taiwanese so just some thoughts.

The "gallery" here is more like a specific term of E-Hentai, not something you would use for gallery apps (e.g. Samsung Gallery).
To distinguish this from other uses, I think it's fine to give it a different translation like "畫廊".
In EhWiki, both "圖庫" and "畫廊" are used, but "相簿" seems not fit here.
image

**Comment by NeKoOuO** [comment]
I also think so

**Comment by bluehomewu** [comment]
> Disclaimer: I'm not Taiwanese so just some thoughts.
> 
> The "gallery" here is more like a specific term of E-Hentai, not something you would use for gallery apps (e.g. Samsung Gallery). To distinguish this from other uses, I think it's fine to give it a different translation like "畫廊". In EHWiki, both "圖庫" and "畫廊" are used, but "相簿" seems not fit here. image

We named "Google Photos" as " Google 相簿", "ASUS Gallery" as "ASUS圖片庫".
I still hold my opinion. I think it is "相簿" is more better.

**Comment by revonateB0T** [comment]
> Disclaimer: I'm not Taiwanese so just some thoughts.
> 
> The "gallery" here is more like a specific term of E-Hentai, not something you would use for gallery apps (e.g. Samsung Gallery). To distinguish this from other uses, I think it's fine to give it a different translation like "畫廊". In EHWiki, both "圖庫" and "畫廊" are used, but "相簿" seems not fit here. image

Why would you consider `相簿` is not appropriate here?
Do you look down on cosplay?
Just kidding XD.
I personally don't make any comments as I'm not Taiwanese.

**Comment by NeKoOuO** [comment]
I personally think that the term '相簿' cannot fully convey the original meaning.

Because that thing in EH is not exactly a '相簿' I think it's a bit inappropriate to translate it as '相簿'

File: app/src/main/res/values-zh-rTW/strings.xml (hunk 19 above)
Comment (FooIbar) [reply app/src/main/res/values-zh-rTW/strings.xml:338]: This should be "相簿" regardless as it's referring to gallery apps.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 23]
```diff
@@ -460,34 +470,36 @@
...
-    <string name="filter_label">隱藏項類型</string>
+    <string name="filter_label">隱藏項型別</string>
     <string name="archive_passwd">壓縮檔密碼</string>
     <string name="archive_need_passwd">壓縮檔需要密碼</string>
     <string name="passwd_wrong">密碼錯誤</string>
     <string name="passwd_cannot_be_empty">密碼不能為空</string>
     <string name="list_tile_thumb_size">詳情模式下縮圖大小</string>
     <string name="account_name">帳號</string>
     <string name="preload_thumb_aggressively">積極地預載入縮圖</string>
+    <string name="animate_items">列表專案動畫</string>
```
Comment (FooIbar) [inline app/src/main/res/values-zh-rTW/strings.xml:489]: "專案" looks suspicious.

Comment (5idereal) [reply app/src/main/res/values-zh-rTW/strings.xml:489]: Should be "列表項目動畫".

**Comment by 5idereal** [comment]
Why can't we keep "圖庫" as the translation? I don't think it needs to be changed. As @FooIbar says, "相簿" doesn't fit here, and "畫廊" is the Simplified Chinese term.

File: app/src/main/res/values-zh-rTW/strings.xml (hunk 19 above)
Comment (NeKoOuO) [reply app/src/main/res/values-zh-rTW/strings.xml]: not sure 337

**Comment by NeKoOuO** [comment]
> Why can't we keep "圖庫" as the translation? I don't think it needs to be changed. As @FooIbar says, "相簿" doesn't fit here, and "畫廊" is the Simplified Chinese term.

Okay I'll change it later

**Comment by NeKoOuO** [comment]
Done

File: app/src/main/res/values-zh-rTW/strings.xml (hunk 19 above)
Comment (FooIbar) [reply app/src/main/res/values-zh-rTW/strings.xml]: 337 is also referring to gallery apps if you look at the original text.
```
Please hide your gallery apps away from other people
```

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 24]
```diff
@@ -119,6 +119,7 @@
     <string name="rate_successfully">評分成功</string>
     <string name="rate_failed">評分失敗</string>
     <string name="no_torrents">沒有種子</string>
+      <string name="no_current_torrents">沒有當前版本圖庫的種子</string>
```
Comment (FooIbar) [inline app/src/main/res/values-zh-rTW/strings.xml:122]: Redundant spaces.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 25]
```diff
@@ -362,7 +371,7 @@
     <string name="settings_advanced_backup_favorite_success">備份收藏列表成功</string>
     <string name="settings_advanced_backup_favorite_failed">備份收藏列表失敗</string>
     <string name="settings_about">關於</string>
-    <string name="settings_about_declaration_summary">EhViewer 與 E-Hentai.org 無任何聯繫</string>
+    <string name="settings_about_declaration_summary">EhViewer 與 E-Hentai.org 無任何聯絡</string>
```
Comment (FooIbar) [inline app/src/main/res/values-zh-rTW/strings.xml:374]: We want to express `association` rather than `communication` here, is `聯絡` proper?

**Review by FooIbar — APPROVED** [review]
LGTM, but someone else must approve.

**Review by revonateB0T — APPROVED** [review]

**Review by 5idereal — CHANGES_REQUESTED** [review]
Please revert the changes for line 501 and 481.

**Comment by NeKoOuO** [comment]
Done

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 26]
```diff
@@ -409,8 +418,8 @@
     <string name="download_archive_started">開始下載壓縮檔</string>
     <string name="download_archive_failure">下載失敗</string>
     <string name="archive_free">免費</string>
-    <string name="archive_original">原始</string>
-    <string name="archive_resample">重新採樣</string>
+    <string name="archive_original">原始檔案</string>
+    <string name="archive_resample">重新取樣檔案</string>
```
Comment (FooIbar) [inline app/src/main/res/values-zh-rTW/strings.xml:422]: These should be as short as possible because the button has limited width, see #1085

**Comment by NeKoOuO** [comment]
Done

**Comment by FooIbar** [comment]
Thank you! And thanks to everyone for reviewing!

**Pull request merged**